client/components/data/query-ticket-support-configuration/index.jsx

```diff
@@ -0,0 +1,35 @@
+/**
+ * External dependencies
+ */
+import { Component } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import {
+	ticketSupportConfigurationRequest,
+} from 'state/ticket-support/configuration/actions';
+
+import {
+	isRequestingTicketSupportConfiguration,
+} from 'state/help/ticket/selectors';
+
+class QueryTicketSupportConfiguration extends Component {
+	componentWillMount() {
+		if ( ! this.props.isRequestingTicketSupportConfiguration ) {
+			this.props.ticketSupportConfigurationRequest();
+		}
+	}
+
+	render() {
+		return null;
+	}
+}
+
+export default connect(
+	( state ) => ( {
+		isRequestingTicketSupportConfiguration: isRequestingTicketSupportConfiguration( state ),
```

Comment:
Just a minor suggestion, feel free to ignore: Since it's already clear from the name of the component that this is a request for ticket support configuration, the prop could just be named `isRequesting`